From deea9b8c257c49c4dbbcbbed64fd0abd61d0335e Mon Sep 17 00:00:00 2001
From: Eric Ruei <e-ruei1@ti.com>
Date: Sat, 21 Nov 2020 00:07:59 +0530
Subject: [PATCH 16/35] HACK: arm: dts: am57xx: add am57[1,2,4]x-idk-pps.dtso

This commit adds the am57[1,2,4]x-idk-pps.dtso which is extension of the
am57[1,2,4]x-idk.dts. It enables the PPS IO pins (sync/latch) and PTP BC
bindings at the modified AM571x-idk EVM where the USB/LCD Display
are disabled.

Signed-off-by: Eric Ruei <e-ruei1@ti.com>
Signed-off-by: Aaron Kramer <a-kramer@ti.com>
Signed-off-by: Lokesh Vutla <lokeshvutla@ti.com>
---
 arch/arm/boot/dts/am57xx-idk-common.dtsi |   5 +
 arch/arm/boot/dts/dra7-pruss.dtsi        |   7 ++
 arch/arm/boot/dts/ti/Makefile            |   4 +-
 arch/arm/boot/dts/ti/am571x-idk-pps.dtso | 144 +++++++++++++++++++++++
 arch/arm/boot/dts/ti/am572x-idk-pps.dtso | 127 ++++++++++++++++++++
 arch/arm/boot/dts/ti/am574x-idk-pps.dtso | 128 ++++++++++++++++++++
 arch/arm/boot/dts/ti/am57xx-idk-pps.dtso |  30 -----
 arch/arm/boot/dts/ti/dtb-merge.cfg       |   6 +-
 8 files changed, 417 insertions(+), 34 deletions(-)
 create mode 100644 arch/arm/boot/dts/ti/am571x-idk-pps.dtso
 create mode 100644 arch/arm/boot/dts/ti/am572x-idk-pps.dtso
 create mode 100644 arch/arm/boot/dts/ti/am574x-idk-pps.dtso
 delete mode 100644 arch/arm/boot/dts/ti/am57xx-idk-pps.dtso

diff --git a/arch/arm/boot/dts/am57xx-idk-common.dtsi b/arch/arm/boot/dts/am57xx-idk-common.dtsi
index 8863bc823639..1917b808b65e 100644
--- a/arch/arm/boot/dts/am57xx-idk-common.dtsi
+++ b/arch/arm/boot/dts/am57xx-idk-common.dtsi
@@ -190,6 +190,11 @@
 			local-mac-address = [00 00 00 00 00 00];
 		};
 	};
+
+	ptp_bc: ptp_bc {
+		compatible = "ti,am57-bc";
+		status = "disabled";
+	};
 };
 
 &dra7_pmx_core {
diff --git a/arch/arm/boot/dts/dra7-pruss.dtsi b/arch/arm/boot/dts/dra7-pruss.dtsi
index 5d9d5fd9a7b3..e870ad4d2c2f 100644
--- a/arch/arm/boot/dts/dra7-pruss.dtsi
+++ b/arch/arm/boot/dts/dra7-pruss.dtsi
@@ -5,6 +5,13 @@
  * Common PRUSS data for TI AM57xx platforms
  */
 
+/ {
+	aliases {
+		pruss_iep1 = &pruss1_iep;
+		pruss_iep2 = &pruss2_iep;
+	};
+};
+
 &ocp {
 	pruss1_tm: target-module@4b226000 {
 		compatible = "ti,sysc-pruss", "ti,sysc";
diff --git a/arch/arm/boot/dts/ti/Makefile b/arch/arm/boot/dts/ti/Makefile
index 1b4ab66ed3d8..c49803bb3cad 100644
--- a/arch/arm/boot/dts/ti/Makefile
+++ b/arch/arm/boot/dts/ti/Makefile
@@ -27,7 +27,9 @@ dtb-y += am57xx-evm.dtbo \
 	dra71-evm-lcd-auo-g101evn01.0.dtbo \
 	dra71-evm-nand.dtbo \
 	dra76-evm-tfp410.dtbo \
-	am57xx-idk-pps.dtbo
+	am571x-idk-pps.dtbo \
+	am572x-idk-pps.dtbo \
+	am574x-idk-pps.dtbo
 
 ifneq ($(DTC),)
   DTC_PATH=$(dir $(DTC))
diff --git a/arch/arm/boot/dts/ti/am571x-idk-pps.dtso b/arch/arm/boot/dts/ti/am571x-idk-pps.dtso
new file mode 100644
index 000000000000..902eabf4a1d7
--- /dev/null
+++ b/arch/arm/boot/dts/ti/am571x-idk-pps.dtso
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Copyright (C) 2019-2020 Texas Instruments Incorporated - http://www.ti.com/
+ */
+
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/clock/dra7.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/dra.h>
+
+&mac {
+	/* cptr pps1 generator and latch properties */
+	pps_timer = <&timer16>;
+	latch_timer = <&timer15>;
+	pinctrl-names = "pwm_off", "pwm_on", "ref_off", "ref_on",
+			"latch_on", "latch_off";
+	pinctrl-0 = <&pps1_timer16_pwm_off>;
+	pinctrl-1 = <&pps1_timer16_pwm_on>;
+	pinctrl-2 = <&pps1_timer16_pwm2_off>;
+	pinctrl-3 = <&pps1_timer16_pwm2_on>;
+	pinctrl-4 = <&pps1_timer15_latch_on>;
+	pinctrl-5 = <&pps1_timer15_latch_off>;
+
+	pps-enable-gpios = <&gpio1 5 GPIO_ACTIVE_HIGH>;
+	ref-enable-gpios = <&gpio6 19 GPIO_ACTIVE_HIGH>;
+	assigned-clocks = <&l4per3_clkctrl DRA7_L4PER3_TIMER16_CLKCTRL 24>, <&l4per3_clkctrl DRA7_L4PER3_TIMER15_CLKCTRL 24>;
+	assigned-clock-parents = <&abe_giclk_div>, <&abe_giclk_div>;
+};
+
+&dpll_abe_ck {
+	assigned-clocks = <&dpll_abe_ck>;
+	assigned-clock-rates = <50000000>;
+};
+
+&pruss1_iep {
+	pinctrl-names = "sync0_off", "sync0_on", "latch0_off", "latch0_on";
+	pinctrl-0 = <&pr1_edc_sync0_off>;
+	pinctrl-1 = <&pr1_edc_sync0_on>;
+	pinctrl-2 = <&pr1_edc_latch0_off>;
+	pinctrl-3 = <&pr1_edc_latch0_on>;
+};
+
+&pruss2_iep {
+	pinctrl-names = "sync0_off", "sync0_on", "latch0_off", "latch0_on";
+	pinctrl-0 = <&pr2_edc_sync0_off>;
+	pinctrl-1 = <&pr2_edc_sync0_on>;
+	pinctrl-2 = <&pr2_edc_latch0_off>;
+	pinctrl-3 = <&pr2_edc_latch0_on>;
+};
+
+&dra7_pmx_core {
+	pps1_timer16_pwm_on: pps1_timer16_pwm_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x36a0, PIN_OUTPUT | MUX_MODE10)
+		>;
+	};
+
+	pps1_timer16_pwm_off: pps1_timer16_pwm_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x36a0, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+
+	pps1_timer16_pwm2_on: pps1_timer16_pwm2_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3680, PIN_OUTPUT | MUX_MODE7)
+		>;
+	};
+
+	pps1_timer16_pwm2_off: pps1_timer16_pwm2_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3680, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+
+	pps1_timer15_latch_on: pps1_timer15_latch_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3684, PIN_INPUT | MUX_MODE7)
+		>;
+	};
+
+	pps1_timer15_latch_off: pps1_timer15_latch_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3684, PIN_INPUT | MUX_MODE15)
+		>;
+	};
+
+	pr1_edc_sync0_on: pr1_edc_sync0_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3724, PIN_OUTPUT | MUX_MODE13)
+		>;
+	};
+
+	pr1_edc_sync0_off: pr1_edc_sync0_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3724, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+
+	pr1_edc_latch0_on: pr1_edc_latch0_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3694, PIN_INPUT | MUX_MODE12)
+		>;
+	};
+
+	pr1_edc_latch0_off: pr1_edc_latch0_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3694, PIN_INPUT | MUX_MODE15)
+		>;
+	};
+
+	pr2_edc_sync0_on: pr2_edc_sync0_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3728, PIN_OUTPUT | MUX_MODE13)
+		>;
+	};
+
+	pr2_edc_sync0_off: pr2_edc_sync0_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3728, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+
+	pr2_edc_latch0_on: pr2_edc_latch0_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3698, PIN_INPUT | MUX_MODE12)
+		>;
+	};
+
+	pr2_edc_latch0_off: pr2_edc_latch0_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3698, PIN_INPUT | MUX_MODE15)
+		>;
+	};
+};
+
+&ptp_bc {
+	pps-sel0-gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
+	pps-sel1-gpios = <&gpio6 7 GPIO_ACTIVE_HIGH>;
+
+	status = "okay";
+};
diff --git a/arch/arm/boot/dts/ti/am572x-idk-pps.dtso b/arch/arm/boot/dts/ti/am572x-idk-pps.dtso
new file mode 100644
index 000000000000..c5c207ec2415
--- /dev/null
+++ b/arch/arm/boot/dts/ti/am572x-idk-pps.dtso
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Copyright (C) 2019-2020 Texas Instruments Incorporated - http://www.ti.com/
+ */
+
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/clock/dra7.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/dra.h>
+
+&mac {
+	/* cptr pps1 generator and latch properties */
+	pps_timer = <&timer16>;
+	latch_timer = <&timer15>;
+	pinctrl-names = "pwm_off", "pwm_on", "ref_off", "ref_on",
+			"latch_on", "latch_off";
+	pinctrl-0 = <&pps1_timer16_pwm_off>;
+	pinctrl-1 = <&pps1_timer16_pwm_on>;
+	pinctrl-2 = <&pps1_timer16_pwm2_off>;
+	pinctrl-3 = <&pps1_timer16_pwm2_on>;
+	pinctrl-4 = <&pps1_timer15_latch_on>;
+	pinctrl-5 = <&pps1_timer15_latch_off>;
+
+	pps-enable-gpios = <&gpio1 5 GPIO_ACTIVE_HIGH>;
+	ref-enable-gpios = <&gpio6 19 GPIO_ACTIVE_HIGH>;
+	assigned-clocks = <&l4per3_clkctrl DRA7_L4PER3_TIMER16_CLKCTRL 24>, <&l4per3_clkctrl DRA7_L4PER3_TIMER15_CLKCTRL 24>;
+	assigned-clock-parents = <&abe_giclk_div>, <&abe_giclk_div>;
+};
+
+&dpll_abe_ck {
+	assigned-clocks = <&dpll_abe_ck>;
+	assigned-clock-rates = <50000000>;
+};
+
+&pruss2_iep {
+	pinctrl-names = "sync0_off", "sync0_on", "latch0_off", "latch0_on",
+			"sync1_off", "sync1_on";
+	pinctrl-0 = <&pr2_edc_sync0_off>;
+	pinctrl-1 = <&pr2_edc_sync0_on>;
+	pinctrl-2 = <&pr2_edc_latch0_off>;
+	pinctrl-3 = <&pr2_edc_latch0_on>;
+	pinctrl-4 = <&pr2_edc_sync1_off>;
+	pinctrl-5 = <&pr2_edc_sync1_on>;
+};
+
+&dra7_pmx_core {
+	pps1_timer16_pwm_on: pps1_timer16_pwm_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x36a0, PIN_OUTPUT | MUX_MODE10)
+		>;
+	};
+
+	pps1_timer16_pwm_off: pps1_timer16_pwm_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x36a0, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+
+	pps1_timer16_pwm2_on: pps1_timer16_pwm2_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3680, PIN_OUTPUT | MUX_MODE7)
+		>;
+	};
+
+	pps1_timer16_pwm2_off: pps1_timer16_pwm2_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3680, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+
+	pps1_timer15_latch_on: pps1_timer15_latch_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3684, PIN_INPUT | MUX_MODE7)
+		>;
+	};
+
+	pps1_timer15_latch_off: pps1_timer15_latch_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3684, PIN_INPUT | MUX_MODE15)
+		>;
+	};
+
+	pr2_edc_sync0_on: pr2_edc_sync0_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35f8, PIN_OUTPUT | MUX_MODE10)
+		>;
+	};
+
+	pr2_edc_sync0_off: pr2_edc_sync0_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35f8, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+
+	pr2_edc_latch0_on: pr2_edc_latch0_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35f0, PIN_INPUT | MUX_MODE10)
+		>;
+	};
+
+	pr2_edc_latch0_off: pr2_edc_latch0_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35f0, PIN_INPUT | MUX_MODE15)
+		>;
+	};
+
+	pr2_edc_sync1_on: pr2_edc_sync1_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35fc, PIN_OUTPUT | MUX_MODE10)
+		>;
+	};
+
+	pr2_edc_sync1_off: pr2_edc_sync1_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35fc, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+};
+
+&ptp_bc {
+	pps-sel0-gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
+	pps-sel1-gpios = <&gpio6 7 GPIO_ACTIVE_HIGH>;
+
+	status = "okay";
+};
diff --git a/arch/arm/boot/dts/ti/am574x-idk-pps.dtso b/arch/arm/boot/dts/ti/am574x-idk-pps.dtso
new file mode 100644
index 000000000000..841fb72df3d9
--- /dev/null
+++ b/arch/arm/boot/dts/ti/am574x-idk-pps.dtso
@@ -0,0 +1,128 @@
+
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Copyright (C) 2019-2020 Texas Instruments Incorporated - http://www.ti.com/
+ */
+
+/dts-v1/;
+/plugin/;
+
+#include <dt-bindings/clock/dra7.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/pinctrl/dra.h>
+
+&mac {
+	/* cptr pps1 generator and latch properties */
+	pps_timer = <&timer16>;
+	latch_timer = <&timer15>;
+	pinctrl-names = "pwm_off", "pwm_on", "ref_off", "ref_on",
+			"latch_on", "latch_off";
+	pinctrl-0 = <&pps1_timer16_pwm_off>;
+	pinctrl-1 = <&pps1_timer16_pwm_on>;
+	pinctrl-2 = <&pps1_timer16_pwm2_off>;
+	pinctrl-3 = <&pps1_timer16_pwm2_on>;
+	pinctrl-4 = <&pps1_timer15_latch_on>;
+	pinctrl-5 = <&pps1_timer15_latch_off>;
+
+	pps-enable-gpios = <&gpio1 5 GPIO_ACTIVE_HIGH>;
+	ref-enable-gpios = <&gpio6 19 GPIO_ACTIVE_HIGH>;
+	assigned-clocks = <&l4per3_clkctrl DRA7_L4PER3_TIMER16_CLKCTRL 24>, <&l4per3_clkctrl DRA7_L4PER3_TIMER15_CLKCTRL 24>;
+	assigned-clock-parents = <&abe_giclk_div>, <&abe_giclk_div>;
+};
+
+&dpll_abe_ck {
+	assigned-clocks = <&dpll_abe_ck>;
+	assigned-clock-rates = <50000000>;
+};
+
+&pruss2_iep {
+	pinctrl-names = "sync0_off", "sync0_on", "latch0_off", "latch0_on",
+			"sync1_off", "sync1_on";
+	pinctrl-0 = <&pr2_edc_sync0_off>;
+	pinctrl-1 = <&pr2_edc_sync0_on>;
+	pinctrl-2 = <&pr2_edc_latch0_off>;
+	pinctrl-3 = <&pr2_edc_latch0_on>;
+	pinctrl-4 = <&pr2_edc_sync1_off>;
+	pinctrl-5 = <&pr2_edc_sync1_on>;
+};
+
+&dra7_pmx_core {
+	pps1_timer16_pwm_on: pps1_timer16_pwm_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x36a0, PIN_OUTPUT | MUX_MODE10)
+		>;
+	};
+
+	pps1_timer16_pwm_off: pps1_timer16_pwm_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x36a0, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+
+	pps1_timer16_pwm2_on: pps1_timer16_pwm2_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3680, PIN_OUTPUT | MUX_MODE7)
+		>;
+	};
+
+	pps1_timer16_pwm2_off: pps1_timer16_pwm2_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3680, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+
+	pps1_timer15_latch_on: pps1_timer15_latch_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3684, PIN_INPUT | MUX_MODE7)
+		>;
+	};
+
+	pps1_timer15_latch_off: pps1_timer15_latch_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x3684, PIN_INPUT | MUX_MODE15)
+		>;
+	};
+
+	pr2_edc_sync0_on: pr2_edc_sync0_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35f8, PIN_OUTPUT | MUX_MODE10)
+		>;
+	};
+
+	pr2_edc_sync0_off: pr2_edc_sync0_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35f8, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+
+	pr2_edc_latch0_on: pr2_edc_latch0_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35f0, PIN_INPUT | MUX_MODE10)
+		>;
+	};
+
+	pr2_edc_latch0_off: pr2_edc_latch0_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35f0, PIN_INPUT | MUX_MODE15)
+		>;
+	};
+
+	pr2_edc_sync1_on: pr2_edc_sync1_on {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35fc, PIN_OUTPUT | MUX_MODE10)
+		>;
+	};
+
+	pr2_edc_sync1_off: pr2_edc_sync1_off {
+		pinctrl-single,pins = <
+			DRA7XX_CORE_IOPAD(0x35fc, PIN_OUTPUT_PULLDOWN | MUX_MODE15)
+		>;
+	};
+};
+
+&ptp_bc {
+	pps-sel0-gpios = <&gpio1 4 GPIO_ACTIVE_HIGH>;
+	pps-sel1-gpios = <&gpio6 7 GPIO_ACTIVE_HIGH>;
+
+	status = "okay";
+};
diff --git a/arch/arm/boot/dts/ti/am57xx-idk-pps.dtso b/arch/arm/boot/dts/ti/am57xx-idk-pps.dtso
deleted file mode 100644
index d08fb85abff7..000000000000
--- a/arch/arm/boot/dts/ti/am57xx-idk-pps.dtso
+++ /dev/null
@@ -1,30 +0,0 @@
-// SPDX-License-Identifier: GPL-2.0
-/*
- * Copyright (C) 2020 Texas Instruments Incorporated - http://www.ti.com/
- */
-
-/dts-v1/;
-/plugin/;
-
-#include <dt-bindings/clock/dra7.h>
-
-/ {
-  fragment@101 {
-	target-path = "/";
-
-	__overlay__ {
-		pwm16: dmtimer-pwm16 {
-			compatible = "ti,omap-dmtimer-pwm";
-			#pwm-cells = <3>;
-			ti,timers = <&timer16>;
-			assigned-clocks = <&l4per3_clkctrl DRA7_L4PER3_TIMER16_CLKCTRL 24>;
-			assigned-clock-parents = <&abe_giclk_div>;
-		};
-	};
-  };
-};
-
-&dpll_abe_ck {
-	assigned-clocks = <&dpll_abe_ck>;
-	assigned-clock-rates = <50000000>;
-};
diff --git a/arch/arm/boot/dts/ti/dtb-merge.cfg b/arch/arm/boot/dts/ti/dtb-merge.cfg
index 1c77610d2fbd..c85463681638 100644
--- a/arch/arm/boot/dts/ti/dtb-merge.cfg
+++ b/arch/arm/boot/dts/ti/dtb-merge.cfg
@@ -5,14 +5,14 @@ am57xx-evm-reva3-cam-ov10635: am57xx-beagle-x15-revc.dtb am57xx-evm-common.dtbo
 
 am571x-idk-lcd-osd101t2045: am571x-idk.dtb am57xx-idk-osd-lcd-common.dtbo am571x-idk-touchscreen.dtbo lcd-osd101t2045.dtbo
 am571x-idk-lcd-osd101t2587: am571x-idk.dtb am57xx-idk-osd-lcd-common.dtbo am571x-idk-touchscreen.dtbo lcd-osd101t2587.dtbo
-am571x-idk-pps: am571x-idk.dtb am57xx-idk-pps.dtbo
+am571x-idk-pps: am571x-idk.dtb am571x-idk-pps.dtbo
 
 am572x-idk-lcd-osd101t2045: am572x-idk.dtb am57xx-idk-osd-lcd-common.dtbo am572x-idk-touchscreen.dtbo lcd-osd101t2045.dtbo
 am572x-idk-lcd-osd101t2587: am572x-idk.dtb am57xx-idk-osd-lcd-common.dtbo am572x-idk-touchscreen.dtbo lcd-osd101t2587.dtbo
-am572x-idk-pps: am572x-idk.dtb am57xx-idk-pps.dtbo
+am572x-idk-pps: am572x-idk.dtb am572x-idk-pps.dtbo
 
 am574x-idk-lcd-osd101t2587: am574x-idk.dtb am57xx-idk-osd-lcd-common.dtbo am572x-idk-touchscreen.dtbo lcd-osd101t2587.dtbo
-am574x-idk-pps: am574x-idk.dtb am57xx-idk-pps.dtbo
+am574x-idk-pps: am574x-idk.dtb am574x-idk-pps.dtbo
 
 dra71-evm-nand: dra71-evm.dtb dra71-evm-nand.dtbo
 dra71-evm-lcd-auo-g101evn01.0: dra71-evm.dtb dra71-evm-lcd-auo-g101evn01.0.dtbo
-- 
2.17.1

